main.go

```diff
@@ -103,6 +104,8 @@ func argsParse(args []string) (runOptionsType, error) {
 
 	updateCheck := parsing.Bool("check-update", false, "force update check")
 
+	updateInventory := parsing.Bool("send-inventory", false, "force inventory update state")
```

Comment:
Maybe "state" can be removed from the help string, so that it is consistent with "force update check"